package.json

```diff
@@ -44,5 +44,9 @@
     "bs-platform": {
       "optional": true
     }
-  }
+  },
+  "workspaces": [
+    "EXAMPLES",
```

Comment:
This change automatically links the version of `rescript-apollo-client` into the `EXAMPLES` workspace, so that you can use the local repository version instead of relying on a version to be published